src/warc2zim/converter.py

```diff
@@ -362,13 +380,15 @@ def find_icon_and_language(self, content):
                 )
             ):
                 self.favicon_url = urljoin(
-                    self.main_url,
+                    f"{self.main_url_scheme}://{self.main_url}",
```

Comment:
You should be able to detect if the href is a scheme-less (same scheme) URL instead of a path and just recompose reusing the main_url's scheme. That would be more reasonnable

Reply:
`main_url` is always scheme-less in current design